lib/widget/widget.cpp

```diff
@@ -216,6 +218,19 @@ static inline void forEachOverlayScreen(const std::function<bool (const OverlayS
 	cleanupDeletedOverlays();
 }
 
+static inline void forEachOverlayScreen(const std::function<bool (const OverlayScreen& overlay)>& func, bool decreasingZOrder = true)
```

Comment:
Maybe it would be better to have another function called ` forEachOverlayScreenReverse`?